CHANGELOG.md

```diff
@@ -54,7 +54,7 @@
 
 * This library now requires a minimum deployment target of iOS 12.0 or above. iOS 11._x_ is no longer supported. ([#4142](https://github.com/mapbox/mapbox-navigation-ios/pull/4142))
 * This library now requires a minimum Xcode version of 13.1.0 or above. ([#4239](https://github.com/mapbox/mapbox-navigation-ios/pull/4239))
-* MapboxCoreNavigation now requires [MapboxDirections v2.8.0-rc.2](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.8.0-rc.2). ([#4239](https://github.com/mapbox/mapbox-navigation-ios/pull/4239))
+* MapboxCoreNavigation now requires [MapboxDirections v2.8.0](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.8.0). ([#4251](https://github.com/mapbox/mapbox-navigation-ios/pull/4251))
```

Comment:
@S2Ler, would you be able to check 6121f1919f569b5ddc69042a81ad9aec84632c81 and verify whether there are any other incorrect entries in the `main` branch at this point?

Reply:
I can also see that https://github.com/mapbox/mapbox-navigation-ios/pull/4239 is mentioned multiple times when it's not really related to description of the entry. I wonder if it was intentional.

Reply:
Seems like other entries are coming from either Navigation Native or Maps.